### PR TITLE
Fix solution name, implement basic Read methods, misc. cleanup

### LINKED ISSUE #10: S-Frame_Toolkit: fix L-Section create

### LINKED ISSUE #17: S-Frame_Toolkit: Add Support for Timber, Aluminium and Generic sections
#### Description:

Add `Create` and `Read` support for `TimberSection`, `AluminiumSection` and `GenericSection` added in https://github.com/BHoM/BHoM/pull/654

### LINKED ISSUE #19: S-Frame_Toolkit: Solution file name to match repo name
Currently the Solution File name is `S-Frame.sln` rather than `S-Frame_Toolkit.sln` which causes problems with the CI/CD and other associated work.

Please rename the Solution file to match the repository name.

### LINKED ISSUE #21: S-Frame_Toolkit: Implement Read Methods

### LINKED ISSUE #32: S-Frame_Toolkit: Remove Common_Engine reference
#### Broken rules:
All Spatial methods have been moved out of the Common_Engine and deprecated.
See https://github.com/BHoM/BHoM_Engine/issues/1624

#### Suggestions to restore compliance:
The Toolkit should switch to using Spatial_Engine instead.

And unless it's using a non-deprecated method from Common_Engine, the reference should be removed as well.

### PR DESCRIPTION
**Pull request by JosefTaylor**
### Issues addressed by this PR

Closes #19 
Closes #17 
Closes #10 
Closes #21 
Closes #32

Solution file called S-Frame_Toolkit
All other files and folders to SConcrete_xxxxx
Added error messages for timber, aluminum, and generic sections
Added read methods - only implemented material, sectionProperty and SConcreteModel. Further development needed.

### Test files

https://burohappold.sharepoint.com/:u:/s/BHoM/Efc_Ld_DsHNFhgvKEuB07DAB7PSnbEBP9OyILQpMu6ILyA?e=3rkAHN

### CODE REVIEW

**Comment by FraserGreenroyd** [comment]
Hi @JosefTaylor can I ask what the status of this PR is? I can see you added a `do-not-merge` label but no clear reason as to why?

I can see this PR has a few conflicts with the `master` branch too, if this PR is unable to be merged could it be closed and reraised when more time can be dedicated to closing out the issues for it?

**Comment by JosefTaylor** [comment]
Yes, it got parked for the open-sourcing and some compliance issues that took priority. The conflicts are mostly just the copyright headers, I believe. I don't know the right way to merge this, but I believe there should be no substantive conflict. There is a lot of work in this branch so I don't want to lose it - how to do that after closing the PR?

**Comment by FraserGreenroyd** [comment]
In that case @JosefTaylor I would recommend merging `master` into this branch to resolve the conflicts and push that up and get this reviewed ASAP by @IsakNaslundBh and co :smile:

File: SConcrete_Adapter/Types/Comparer.cs [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
 
 ﻿/*
```
Comment (JosefTaylor) [inline SConcrete_Adapter/Types/Comparer.cs:24]: old copyright

**Pull request merged**